spec/features/adventures/user_edits_adventure_spec.rb

```diff
@@ -114,23 +146,20 @@
     expect(page).not_to have_content("Changes saved!")
   end
 
-  scenario "authenticated user fails to edit an adventure" do
-    user = FactoryGirl.create(:user)
-    bucket_list_sign_in(user)
-    adventure = FactoryGirl.create(:adventure, address: nil, user_id: user.id)
-
-    visit edit_adventure_path(adventure)
-    find("input[id$='adventure_name']").set ""
-    click_button "Save It!"
-
-    expect(page).to have_content("Must specify a name and/or address")
-    expect(page).not_to have_content("Changes saved!")
-  end
-
   scenario "authenticated user successfully removes adventure name attribute" do
     user = FactoryGirl.create(:user)
-    bucket_list_sign_in(user)
+    sign_in(user)
+    bucket_list = FactoryGirl.create(
+      :bucket_list,
+      user_id: user.id,
+      title: "Second Bucket List"
+    )
     adventure = FactoryGirl.create(:adventure, user_id: user.id)
+    bucket_list_adventure = FactoryGirl.create(
```

Comment:
Useless assignment to variable - `bucket_list_adventure`. Did you mean `bucket_list`?